packages/admin-test-utils/package.json

```diff
@@ -72,7 +72,7 @@
   },
   "devDependencies": {
     "@reduxjs/toolkit": "1.9.7",
-    "@strapi/pack-up": "workspace:*",
+    "@strapi/pack-up": "4.23.0",
```

Comment:
Just checking since I know you released a 5.0.0 of pack-up.  Is the intent for the major versions to align with Strapi in case we need to add a fix specifically for strapi v4?

Reply:
well v5 of pack-up fails if the types fail, and V4 Strapi has circular dependencies so the types mostly fail in a real world setting, so we can't use v5 pack-up

Reply:
Ok, works for now :this-is-fine: